R/uvhydrograph-fnc.R

```diff
@@ -1,133 +1,141 @@
 
 #'@export
 uvhydrographPlot <- function(data){
-  uv_pts <- getUvHydro(data, "primarySeries" )
-  layout_uvhydro()
-  
-  primary_lbl <- getUvLabel(data, "primarySeries")
-  
-  add_uvhydro_axes(getUvhLims(uv_pts), ylab = primary_lbl, ylog = TRUE)
-  
-  dv_pts <- getUvHydro(data, "derivedSeriesMean" )
-  dv_pts$x = dv_pts$x + 86400/2 # manual shift for now...
-  
-  # discharge measurements and errors
-  if(isSeriesOfType(data, "primarySeries", "Discharge")) {
-    add_q_measurements(data)
-  }
-  
-  add_computed_uv(uv_pts)
-  add_approved_dv(dv_pts)
-  
-  secondary_lbl <- getUvLabel(data, "secondarySeries")
-  tertiary_lbl <- getUvLabel(data, "effectiveShifts")
-  
-  uv2_pts <- getUvHydro(data, "secondarySeries")
-  shift_pts <- getUvHydro(data, "effectiveShifts")
-  
-  secondary_lims <- getUvhLims(uv2_pts)
-  add_uvhydro_axes(secondary_lims, ylab = secondary_lbl, ylog = FALSE)
+  #for pagination by month, get all of the month strings for the primary series
+  all_primary_pts <- getUvHydro(data, "primarySeries" )
+  months <- unique(all_primary_pts$month, incomparables = FALSE)
```

Comment:
they might not be continuous, but they will be in order, and missing a month should be OK